Framework/PythonInterface/mantid/api/src/Exports/Run.cpp

```diff
@@ -243,5 +243,6 @@ void export_Run() {
       .def("__setitem__", &addOrReplaceProperty,
            (arg("self"), arg("name"), arg("value")))
       .def("__copy__", &Mantid::PythonInterface::generic__copy__<Run>)
-      .def("__deepcopy__", &Mantid::PythonInterface::generic__deepcopy__<Run>);
+      .def("__deepcopy__", &Mantid::PythonInterface::generic__deepcopy__<Run>)
+      .def("__eq__", &Run::operator==, arg("self"), arg("other"));
```

Comment:
Also define `__ne__` here and in `Sample`?

Reply:
Hmm, I see that in Python 3 it is just defined as the negation of `__eq__`. Is Python 2 support completely dropped in Mantid?

Reply:
I don't know what the roadmap is for dropping Python 2 support. It should be added maybe added conditionally.

Reply:
Since there is no harm having it also in Python 3 I would not bother with the added complexity of making it conditional.

Reply:
2.7 is now completely dropped as of release 5.0 we don't need `__ne__`